promptsource/templates.py

```diff
@@ -375,7 +375,7 @@ def write_to_file(self) -> None:
         # We only create the folder if a template is written
         if not os.path.exists(self.folder_path):
             os.makedirs(self.folder_path)
-        yaml.dump(self.format_for_dump(), open(self.yaml_path, "w"))
+        yaml.dump(self.format_for_dump(), open(self.yaml_path, "w"), default_style="|")
```

Comment:
This is where we may need a consensus.
`default_style` means it will change everything when updating templates.
While it will be consistent and easier to read for the jinja part, the rest may become more complex.